### PR TITLE
Add docs about using ADAM's Kryo registrator from another Kryo registrator.

### PR DESCRIPTION
**Pull request by fnothaft**
Follow on from @heuermh's #1304.

### CODE REVIEW

File: docs/source/60_building_apps.md [hunk 1]
```diff
@@ -187,7 +187,10 @@ object MyExample {
 ```
 
 Create an Apache Spark configuration `SparkConf` and use it to create a new `SparkContext`.
-The following serialization configuration needs to be present to register ADAM classes.
+The following serialization configuration needs to be present to register ADAM classes. This
+assumes that you do not need to register additional custom
+[Kryo Serializers](https://github.com/EsotericSoftware/kryo). If you do, you will need to
+[write your own registrator that calls the ADAM registrator](#registrator).
```
Comment (heuermh) [inline docs/source/60_building_apps.md]: `[Kyro Serializers]` &rarr; `[Kyro serializers]`

Would this read better as:

The following serialization configuration needs to be present to register ADAM classes. If any additional [Kyro serializers](https://github.com/EsotericSoftware/kryo) need to be registered, (...and why would they need to be?...), [create a registrator that delegates to the ADAM registrator](#registrator).

File: docs/source/60_building_apps.md [hunk 2]
```diff
@@ -249,3 +252,32 @@ $ spark-submit \
 
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
+
+### Writing your own registrator that calls the ADAM registrator {#registrator}
+
+As we do in ADAM, you may want to provide your own Kryo serializer registrator,
```
Comment (heuermh) [inline docs/source/60_building_apps.md]: I hadn't been using "you" in this doc, so this paragraph doesn't use the same voice as above.  I guess I didn't check the other doc pages to see.

File: docs/source/60_building_apps.md [hunk 3]
```diff
@@ -249,3 +252,32 @@ $ spark-submit \
...
+whether to register custom serializers, or as part of a workflow where you
+require all serializers to be registered. In either case, you will need to
+provide your own Kryo registrator. While you can manually register ADAM's
+serializers in your own registrator, it is simpler to call to the ADAM
+registrator from within your registrator. To do this, you would write code
+like this:
+
+```
```
Comment (heuermh) [inline docs/source/60_building_apps.md]: Use ```scala for syntax highlighting

File: docs/source/60_building_apps.md [hunk 4]
```diff
@@ -249,3 +252,32 @@ $ spark-submit \
...
+import com.esotericsoftware.kryo.Kryo
+import org.apache.spark.serializer.KryoRegistrator
+import org.bdgenomics.adam.serialization.ADAMKryoRegistrator
+
+class AvocadoKryoRegistrator extends KryoRegistrator {
```
Comment (heuermh) [inline docs/source/60_building_apps.md]: ```AvocadoKryoRegistrator``` &rarr; ```MyCommandKyroRegistrator``` to match example code above

**Comment by fnothaft** [comment]
Thanks for the review @heuermh! I've rolled your suggestions in.

**Comment by heuermh** [comment]
Looks great, thank you, @fnothaft

**Pull request merged**